kagami/core/supervisor.py

```diff
@@ -35,18 +40,23 @@ def __init__(
                 raw_worker_info[worker_info.worker_addr] = worker_info
         self.registered_workers = raw_worker_info
 
-        # build resource_info
-        raw_resource_info = {}
-        for resource_info in resource_info_list:
-            if resource_info is not None:
-                raw_resource_info[resource_info.name] = resource_info
-
         self.unregistered_worker = []
 
-    # TODO load config from database and configuration file
     # @classmethod
-    # def load(self) -> "Supervisor":
-    #     return Supervisor()
+    # async def load(cls, config: SupervisorConfig) -> "Supervisor":
+    #     # reserve for load from config
+    #     # Load the worker from database (if not init)
+    #     worker_service = WorkerService(session=session_generator())
+    #     workers = await worker_service._session.execute(
+    #         select(Worker)
+    #     )  # TODO alternative database service
```

Comment:
@hedgehog098 Can you write a function to manage database read operations?
For example:
select_all_worker() # list workers
get_worker_by_worker_addr() 
etc.
same as Admin database.

Reply:
ok